### ISSUE TITLE
Difference in Number of Parameters in Paper and in the network

### ISSUE THREAD

**Issue by IamVNIE** [busyyang/COVID-19#2]
The paper says the  number of parameters in COVID-Net is 116.6 Million. However, this keras network has 364.6 Million Parameters, which is more than 3 times the parameter. Any idea why?

**Comment by busyyang** [comment]
Hi, when I implement this repo. I just set the `DWConv3x3`  as normal conv layer. You can shift it to `DepthwiseConv2D(......)`. 
~~~py
from tensorflow.keras.layers import Conv2D, Input, MaxPool2D, add, Flatten, Dense, DepthwiseConv2D


def PEPXModel(input_tensor, filters, name):
    x = Conv2D(filters=filters, kernel_size=(1, 1), activation='relu', name=name + 'FP')(input_tensor)
    x = Conv2D(filters=filters, kernel_size=(1, 1), activation='relu', name=name + 'Expansion')(x)
    x = DepthwiseConv2D(kernel_size=(3, 3), activation='relu', padding='same', name=name + 'DWConv3_3')(x)
    x = Conv2D(filters=filters, kernel_size=(1, 1), activation='relu', name=name + 'SP')(x)
    x = Conv2D(filters=filters, kernel_size=(1, 1), activation='relu', name=name + 'Extension')(x)
    return x
~~~
But even you have done it. The params will be 183.7M, not 116.6 M. I have no idea where is the problem, If you notice something, let's dicuss it.

I have update the [keras_model.py](https://github.com/busyyang/COVID-19/blob/master/keras_model.py) for convenient usage.
